templates/nodejs/issue_template.tpl

```diff
@@ -16,7 +15,7 @@ possible:
   - OS:
   - Node.js version:
   - npm version:
-  - {{lib_pkg_name}} version:
+  - `{{lib_pkg_name}}` version:
```

Comment:
This is so the name of the package shows.

With: `@google-cloud/storage`
Without: @google-cloud/storage

(seriously!)